src/main/java/edu/kis/powp/jobs2d/drivers/factories/TriangleFactory.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers.factories;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.ComplexCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+
+public class TriangleFactory  {
+
+    public static void drawTriangle(Job2dDriver driver) {
```

Comment:
This is not a factory. A factory should return a driver command (not execute it). Missing point 3.1.5.